### DISCUSSION TITLE
Prevent Modal Close on Outside Click

### DISCUSSION THREAD

**Discussion by protoapostoli** [wireui/wireui#657] [Q&A]
Hi, 

Love your work!

Was wondering, is it possible to prevent the modal from closing due to an outside click?
I'd prefer the modal to close when the user presses the close button in the modal, and not when they accidentally click outside of it.

Thanks.

**Comment by Joemires** [comment]
@protoapostoli 
I just made a PR that supports that, check below PR link https://github.com/wireui/wireui/commit/8718b1813753f75c41f448490e278b41bf0d557c

You can now add ```persistence=bool``` to your modal component

**Reply by protoapostoli** [reply]
Awesome, man. This is great!
